website/project/decorators.py

```diff
@@ -329,3 +330,15 @@ def wrapped(*args, **kwargs):
 
     # Return decorator
     return wrapper
+
+
+def dev_only(func):
```

Comment:
Longterm I agree. I suspect this was a stopgap until we released the API, but will ping @brianjgeiger for followup on design rationale. 

Even if we keep this decorator in general, when should it stop being applied to settings pages for oauth platform application registration?